includes/gateways/stripe/includes/give-stripe-helpers.php

```diff
@@ -52,7 +52,7 @@ function give_stripe_get_secret_key( $form_id = 0 ) {
 function give_stripe_get_connected_account_id( $form_id = 0 ) {
 	$default_account = give_stripe_get_default_account( $form_id );
 
-	return trim( $default_account['account_id'] );
+	return isset( $default_account['account_id'] ) ? trim( $default_account['account_id'] ) :  false;
```

Comment:
@DevinWalker always returns only one data type.  In the above care, this function must return an empty string.

Reply:
I will take care of this

Reply:
fixed: https://github.com/impress-org/givewp/compare/0e92af6d0aac...42aa54620744